velox/common/base/SpillConfig.h

```diff
@@ -69,10 +70,17 @@ struct SpillConfig {
   /// the next level of recursive spilling.
   int32_t joinSpillLevel(uint8_t startBitOffset) const;
 
+  /// TODO: Remove above method
+  int32_t spillLevel(uint8_t startBitOffset, uint8_t partitionBits) const;
+
   /// Checks if the given 'startBitOffset' has exceeded the max hash join
   /// spill limit.
   bool exceedJoinSpillLevelLimit(uint8_t startBitOffset) const;
 
+  /// TODO: Remove above method
+  bool exceedSpillLevelLimit(uint8_t startBitOffset, uint8_t partitionBits)
```

Comment:
Can you replace in this PR? It doesn't break anything? Thanks!

Reply:
Fixed